src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1122,6 +1122,28 @@ void M2(string value1, string value2)
 //]
     }}
 }}
+";
+
+        private static readonly string s_preferParameterNullChecking = $@"
+using System;
+
+class Customer
+{{
+//[
+    // {ServicesVSResources.Prefer_colon}
+    void M1(string value!!)
+    {{
+    }}
+//]
+//[
+    // {ServicesVSResources.Over_colon}
+    void M2(string value)
```

Comment:
```suggestion
    void M1(string value)
```

Reply:
It's not clear to me that this is the right change to make, since other samples in this file use the `M1`/`M2` naming convention.

Reply:
The intent is that the other section is what happans _after_ this option is applied so as the user flips the option back and forth they can see the difference

Reply:
yes.  this needs to change to M1